probe_scraper/runner.py

```diff
@@ -2,28 +2,35 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import os
-import json
+from collections import defaultdict
+from dateutil.tz import tzlocal
+import argparse
 import datetime
 import errno
-from dateutil.tz import tzlocal
+import json
+import os
 import tempfile
-import argparse
-from collections import defaultdict
+import traceback
 
-import scraper
+from emailer import send_ses
+from parsers.events import EventsParser
 from parsers.histograms import HistogramsParser
 from parsers.scalars import ScalarsParser
-from parsers.events import EventsParser
-import transform_revisions
+from scrapers import git_scraper, moz_central_scraper
+from schema import And, Optional, Schema
 import transform_probes
+import transform_revisions
 
 
 class DummyParser:
     def parse(self, files):
         return {}
 
 
+FROM_EMAIL = "telemetry-alerts@mozilla.com"
+DEFAULT_TO_EMAIL = "dev-telemetry-alerts@mozilla.com"
```

Comment:
Hm, is that a good default email list? Do we have anything more suitable?
This would overload that list usage even more.

Reply:
I filter those emails pretty aggressively, if people want to ignore them they can always just send `subject contains "Probe Scraper"` to trash. I'm not sure of a better alternative, did you have any ideas?

Reply:
Right, that's maybe better as a future point.
Can you give Chris a heads-up on this?